### PR TITLE
more cves

### PR DESCRIPTION
**Pull request by kurtseifried**

### CODE REVIEW

File: 2018/1000xxx/CVE-2018-1000673.json [hunk 1]
```diff
@@ -0,0 +1 @@
... ]                                 }                             }                         ]                     },                     "vendor_name": "okular"                 }             ]         }     },     "data_format": "MITRE",     "data_type": "CVE",     "data_version": "4.0",     "description": {         "description_data": [             {                 "lang": "eng",                 "value": "okular version 1.5 and earlier contains a Directory Traversal vulnerability in function  \"unpackDocumentArchive(...)\" in \"core/document.cpp\" that can result in Arbitrary file creation on the user workstation.  This attack appear to be exploitable via he victim must open a specially crafted Okular archive."             }         ]     },     "problemtype": {         "problemtype_data": [             {                 "description": [                     {                         "lang": "eng",                         "value": "Directory Traversal"                     }                 ]             }         ]     },     "references": {         "reference_data": [             {                 "url": "https://bugs.kde.org/show_bug.cgi?id=398096"             }         ]     } }
```
Comment (cve-team) [inline 2018/1000xxx/CVE-2018-1000673.json]: The version "1.5 and earlier" is listed in the description and "version_value" but the reference mentions this is fixed in 18.08.1. The supporting okular references/commits also mention 18.08 for the vulnerability.

File: 2018/1000xxx/CVE-2018-1000671.json [hunk 2]
```diff
@@ -0,0 +1 @@
...pa"                 }             ]         }     },     "data_format": "MITRE",     "data_type": "CVE",     "data_version": "4.0",     "description": {         "description_data": [             {                 "lang": "eng",                 "value": "sympa version 4.0a5 and later contains a CWE-601: URL Redirection to Untrusted Site ('Open Redirect')  vulnerability in The \"referer\" parameter of the wwsympa.fcgi login action. that can result in Open redirection and reflected  XSS via data URIs. This attack appear to be exploitable via Victim's browser must follow a URL supplied by the attacker. This  vulnerability appears to have been fixed in none available."             }         ]     },     "problemtype": {         "problemtype_data": [             {                 "description": [                     {                         "lang": "eng",                         "value": "CWE-601: URL Redirection to Untrusted Site ('Open Redirect')"                     }                 ]             }         ]     },     "references": {         "reference_data": [             {                 "url": "https://github.com/sympa-community/sympa/issues/268"             }         ]     } }
```
Comment (cve-team) [inline 2018/1000xxx/CVE-2018-1000671.json]: I couldn't verify the version information in this one (version 4.0a5 and later). Looking at the releases, I only found ones in the 6.x range. Is the 4.0a5 accurate?

File: 2018/1000xxx/CVE-2018-1000670.json [hunk 3]
```diff
@@ -0,0 +1 @@
...ion_data": [             {                 "lang": "eng",                 "value": "KOHA Library System version Priorto version 16.05 contains a Cross Site Scripting (XSS) vulnerability  in Multiple fields on multiple pages including /cgi-bin/koha/acqui/supplier.pl?op=enter ,  /cgi-bin/koha/circ/circulation.pl?borrowernumber=[number] , /cgi-bin/koha/serials/subscription-add.pl that can result in  Privillege escalation by taking control of higher privilleged users browser sessions. This attack appear to be exploitable via  Victims must be socially engineered to visit a vulnerable webpage containing malicious payload. This vulnerability appears to  have been fixed in 16.05.17 and later."             }         ]     },     "problemtype": {         "problemtype_data": [             {                 "description": [                     {                         "lang": "eng",                         "value": "Cross Site Scripting (XSS)"                     }                 ]             }         ]     },     "references": {         "reference_data": [             {                 "url": "https://bugs.koha-community.org/bugzilla3/show_bug.cgi?id=19086"             }         ]     } }
```
Comment (cve-team) [inline 2018/1000xxx/CVE-2018-1000670.json]: Looking at the comments in the reference provided, it seems like this also applies to 16.11.x (up until 16.11.12) and 17.05.x (up until 17.05.04). The only version mentioned in the version value and description is 16.05 (16.05.17). Should the others above be included as well?

File: 2018/1000xxx/CVE-2018-1000669.json [hunk 4]
```diff
@@ -0,0 +1 @@
...data_format": "MITRE",     "data_type": "CVE",     "data_version": "4.0",     "description": {         "description_data": [             {                 "lang": "eng",                 "value": "KOHA Library System version prior to 16.05 contains a Cross ite Request Forgery (CSRF) vulnerability in /cgi-bin/koha/members/paycollect.pl Parameters affected: borrowernumber, amount, amountoutstanding, paid that can result in Attackers can mark payments as paid for certain users on behalf of Administrators. This attack appear to be exploitable via The victim must be socially engineered into clicking a link, usually via email.. This vulnerability appears to have been fixed in 16.05.18."             }         ]     },     "problemtype": {         "problemtype_data": [             {                 "description": [                     {                         "lang": "eng",                         "value": "Cross ite Request Forgery (CSRF)"                     }                 ]             }         ]     },     "references": {         "reference_data": [             {                 "url": "https://bugs.koha-community.org/bugzilla3/show_bug.cgi?id=19117"             }         ]     } }
```
Comment (cve-team) [inline 2018/1000xxx/CVE-2018-1000669.json]: Looking at the comments in the reference provided, it seems like this also applies to 16.11.x (up until 16.11.13) and 17.05.x (up until 17.05.05), as well as a fix in 17.11. The only version mentioned in the version value and description is 16.05 (16.05.18). Should the others above be included as well?

**Comment by cve-team** [comment]
CVE-2018-1000662 and CVE-2018-1000672 have been rejected so this pull request will represent a merge conflict with those included. Once the other comments are addressed/fixed, just create a new PR without those ID's included. I'm closing this one for now. Thanks!

File: 2018/1000xxx/CVE-2018-1000673.json (hunk 1 above)
Comment (kurtseifried) [reply 2018/1000xxx/CVE-2018-1000673.json]: REJECT'ed this, replaced okular with CVE-2018-1000801

File: 2018/1000xxx/CVE-2018-1000671.json (hunk 2 above)
Comment (kurtseifried) [reply 2018/1000xxx/CVE-2018-1000671.json]: this is now in https://github.com/CVEProject/cvelist/pull/1022

**Pull request closed without merging**